IdentityCore/tests/automation/ui_app_lib/ui_common/ios/MainAutomation.storyboard

```diff
@@ -1,11 +1,8 @@
 <?xml version="1.0" encoding="UTF-8"?>
```

Comment:
This pull request does not update changelog.txt.

Please consider if this change would be noticeable to a partner or user and either update changelog.txt or resolve this conversation.
<!-- changelog-comment -->